### PR TITLE
Update how-can-caller-id-be-used-in-your-organization.md

### PR DESCRIPTION
**Pull request by stvollet**
adding "teams calling plans , teams direct routing" for clarity on the available pstn connectivity option. thank you

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
Please note that the  product names are Phone System Direct Routing and Microsoft Calling Plans.  Make those changes and then I will merge the PR.  Thank you.

File: Teams/how-can-caller-id-be-used-in-your-organization.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ description: "Caller ID can be controlled for both inbound and outbound calls fo
 Caller ID can be controlled for both inbound and outbound calls for Phone System users by using a policy called CallingLineIdentity.
   
 The caller ID functionality is available to all Phone System users regardless of PSTN connectivity:
+
+- Teams Calling Plans
```
Comment (JohanFreelancer9) [inline Teams/how-can-caller-id-be-used-in-your-organization.md:33]:
```suggestion
- Microsoft Calling Plans 
```

File: Teams/how-can-caller-id-be-used-in-your-organization.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ description: "Caller ID can be controlled for both inbound and outbound calls fo
 Caller ID can be controlled for both inbound and outbound calls for Phone System users by using a policy called CallingLineIdentity.
   
 The caller ID functionality is available to all Phone System users regardless of PSTN connectivity:
+
+- Teams Calling Plans 
+
+- Teams Direct Routing
```
Comment (JohanFreelancer9) [inline Teams/how-can-caller-id-be-used-in-your-organization.md:35]:
```suggestion
- Phone System Direct Routing 
```

**Pull request merged**